corehq/apps/userreports/pillow.py

```diff
@@ -432,9 +433,9 @@ def checkpoint_updated(self):
                 break
 
         for domain, duration in top_half_domains.items():
-            datadog_histogram('commcare.change_feed.ucr_slow_log', duration, tags=[
-                'domain:{}'.format(domain)
-            ])
+            metrics_counter('commcare.change_feed.ucr_slow_log', duration, tags={
+                'domain': domain
+            })
```

Comment:
this isn't referenced on any DD dashboards but seems useful to have around. Changed to a straight counter which I think still serves the purpose.